scripts/openapi/salesflare/metadata/main.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"log/slog"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/internal/datautils"
+	"github.com/amp-labs/connectors/internal/goutils"
+	"github.com/amp-labs/connectors/internal/metadatadef"
+	"github.com/amp-labs/connectors/internal/staticschema"
+	"github.com/amp-labs/connectors/scripts/openapi/salesflare/internal/files"
+	"github.com/amp-labs/connectors/tools/fileconv/api3"
+	"github.com/amp-labs/connectors/tools/scrapper"
+)
+
+// nolint:gochecknoglobals
+var (
+	ignoreEndpoints = []string{
+		"/settings/ai",
+		"/me",
+	}
+)
+
+func main() {
+	schemas := staticschema.NewMetadata[staticschema.FieldMetadataMapV2]()
+	registry := datautils.NamedLists[string]{}
+
+	for _, object := range Objects() {
+		objectName := object.ObjectName
+
+		if object.Problem != nil {
+			slog.Error("schema not extracted",
```

Comment:
Avoid calling slog directory, use `logging.Logger(ctx)` - that way we can add attributes to the log messages (such as customer id) from the server.